install_pim/manual/system_requirements/system_install_ubuntu_1804.rst

```diff
@@ -36,10 +36,10 @@ Now is the time to install what has been configured in the step before:
 
 When installing MySQL 8.0, you'll have to choose the authentication method. Please select *Use Legacy Authentication Method* as the *Strong Password Encryption* is not yet supported by Akeneo PIM.
 
-PHP 7.3
+PHP 7.4
 *******
 
-As Ubuntu 18.04 only provides PHP 7.2, we need to use `Ondrej Sury <https://deb.sury.org/>`_ packages to install PHP 7.3..
+As Ubuntu 18.04 only provides PHP 7.2, we need to use `Ondrej Sury <https://deb.sury.org/>`_ packages to install PHP 7.4..
```

Comment:
we might want to update the doc for a newer version of ubuntu as well (e.g Ubuntu 20.04 LTS), but that's another story...